README.markdown

```diff
@@ -1,5 +1,31 @@
 # What is GitX?
 
+GitX is an OS X (MacOS) Native graphical client for the `git` version control system.
+
+
+GitX has a long history of various branches and versions maintained by various people over the years.  This github org & repo are an attempt to consolidate and move forward with a current, common, community-maintained version.   See discussion at [**Forking, and plans for the future of GitX**](https://github.com/gitx/gitx.github.io/issues/1).
+
+### How to Build & Install:
+
+For build instructions, see [the wiki page](https://github.com/gitx/gitx/wiki/Build-instructions)
+
+<br>
+<br>
+<br>
+<br>
+<br>
+
+
+## [Old README]
+
+Below here is the contents of the README from the @rowanj fork of GitX.  Much of the information there is obsolete.
```

Comment:
I agree here too.  But felt presumptuous to completely wipe out the README of somebody else's project.  Again just trying to do the bare minimum to have a more current (and not misleading) README